terraform/helm/aptos-node/files/configs/fullnode.yaml

```diff
@@ -26,13 +26,14 @@ full_node_networks:
 - network_id: "public"
   discovery_method: "onchain"
   listen_address: "/ip4/0.0.0.0/tcp/6182"
+  identity:
+    type: "from_file"
+    path: "/opt/aptos/genesis/validator-full-node-identity.yaml"
```

Comment:
We can also apply this to the VFN network in the future, and then both sides will only be able to talk to each other regardless of tunneling :)  But, let's not rock the boat on it right now